clang/lib/AST/Decl.cpp

```diff
@@ -1605,8 +1610,7 @@ void NamedDecl::printNestedNameSpecifier(raw_ostream &OS,
       if (ND->isAnonymousNamespace()) {
         OS << (P.MSVCFormatting ? "`anonymous namespace\'"
                                 : "(anonymous namespace)");
-      }
-      else
+      } else
```

Comment:
Unrelated formatting change... I'm OK with it, but @bader might not be.

Reply:
Oh :/ That's from code I removed (from an earlier commit) when doing suggested refactoring. I think it is okay since its 'correct' but I can change it if you'd prefer @bader

Reply:
I'm okay with this change as it's aligned with the LLVM coding guidelines.